### PR TITLE
README.md: add app store badge and link

### PR DESCRIPTION
**Pull request by RandyMcMillan**
Remove Testflight link
Remove reference to bird app
Clickable images to App Store

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
 
 # damus
+#### A twitter-like [nostr][nostr] client for iPhone, iPad and MacOS.
+<HR>
+<center>
+<a href="https://apps.apple.com/us/app/damus/id1628663131" rel="damus nostr app" style="width: 50%; height: 50%">
```
Comment (RandyMcMillan) [inline README.md:6]: images are clickable for easy navigation to the App Store.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+![Damus](./ss.png)
+</a>
+<a href="https://apps.apple.com/us/app/damus/id1628663131" rel="damus nostr app" style="width: 100%; height: 100%">
```
Comment (RandyMcMillan) [inline README.md:9]: images are clickable for easy navigation to the App Store.

File: README.md [hunk 3]
```diff
@@ -137,3 +139,5 @@ First contributors:
 ### git log bot
 
 npub1fjtdwclt9lspjy8huu3qklr7eklp5uq90u6yh8mec290pqxraccqlufnas
+
+[nostr]: https://github.com/fiatjaf/nostr
```
Comment (RandyMcMillan) [inline README.md:143]: moved markdown ref link to bottom of README.md

**Comment by RandyMcMillan** [comment]
App images and App Store badge are clickable for easy navigation to the App Store.

Screen Shot 2023-02-04 at 2 30 49 PM

**Comment by jb55** [comment]
awesome!

**Comment by RandyMcMillan** [comment]
your welcome brother!!

**Comment by jb55** [comment]
Let's not mention twitter tho

**Comment by RandyMcMillan** [comment]
ok - i was thinking that as well.

**Comment by RandyMcMillan** [comment]
Updated: [README.md](https://github.com/damus-io/damus/blob/56adda24c1d64042341d49173755dcf4a3517fff/README.md)

**Comment by RandyMcMillan** [comment]
commit: 56adda24c1d64042341d49173755dcf4a3517fff
Remote Desktop Picture February 4, 2023 at 4 42 44 PM EST

File: README.md [hunk 4]
```diff
@@ -138,3 +134,11 @@ First contributors:
...
+[damus]: https://damus.io
```
Comment (RandyMcMillan) [inline README.md:139]: group links together

File: README.md [hunk 5]
```diff
@@ -1,28 +1,24 @@
-[![Run Test Suite](https://github.com/damus-io/damus/actions/workflows/run-tests.yaml/badge.svg?branch=master)](https://github.com/damus-io/damus/actions/workflows/run-tests.yaml)
+# Damus - The social network you control.
```
Comment (RandyMcMillan) [inline README.md:1]: remove reference to bird app.

**Comment by jb55** [comment]
I don't think style works in github, you need width and height attrs

**Pull request closed without merging**